velox/docs/monthly-updates/june-2022.rst

```diff
@@ -0,0 +1,61 @@
+********************
+June 2022 Update
+********************
+
+Documentation
+=============
+
+* Add documentation for :ref:`complex types writers<outputs-write>`.
+
+Core Library
+============
+
+* Add support for IntervalDayTime type.
+* Allow cast between Date type and Timestamp type.
+* Add fast path to :func:`zip` when all Arrays are flat and the same size.
+* Add GroupId operator.
```

Comment:
Perhaps, add a brief explanation of what this operator is used for and add a link to https://facebookincubator.github.io/velox/develop/operators.html#groupidnode.

`Add GroupIdNode and GroupId operator to support aggregations over grouping sets.`